Test/TestConfigurationUtils.ps1

```diff
@@ -197,6 +197,45 @@ function Wait-RemoteInterfaceIP {
     } | Out-Null
 }
 
+function Test-IfVmSwitchExist {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session,
```

Comment:
pls split into multiple lines like in `Assert-VmSwitchInitialized`

Reply:
Fixed